legion/engine/physics/physx/systems/physx_physics_system.cpp

```diff
@@ -104,14 +231,14 @@ namespace legion::physics
             PS::foundation = PxCreateFoundation(PX_PHYSICS_VERSION,
                 PS::defaultAllocator, PS::defaultErrorCallback);
 
-            PS::pvd = PxCreatePvd(*PS::foundation);
-            PxPvdTransport* transport = PxDefaultPvdSocketTransportCreate(pvdHost, defaultPVDListeningPort, defaultPVDHostTimeout);
-            PS::pvd->connect(*transport, PxPvdInstrumentationFlag::eDEBUG);
-
             PS::dispatcher = PxDefaultCpuDispatcherCreate(0); //deal with multithreading later on
 
+            #ifdef _DEBUG
+            PS::debugger.initializeDebugger(PS::foundation, DebuggerWrapper::transport_mode::none);
+            #endif
```

Comment:
do you never wanna do this in release debug? cuz if you do, then you might wanna use `LEGION_VALIDATE` or `LEGION_DEBUG`. since we can make sure those are defined even when compiling for release. to debug the release version.

Reply:
pvd output is disabled on release libs, but I'll use LEGION_DEBUG anyway for consistency